server/slack-bot.js

```diff
@@ -41,14 +41,26 @@ This bot demonstrates many of the core features of Botkit:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 require('isomorphic-fetch');
 const queryString = require('query-string');
+const port = process.env.PORT || process.env.VCAP_APP_PORT || 3000;
 const slackBotToken = process.env.SLACK_BOT_TOKEN || 
                       process.env.VCAP_APP_SLACK_BOT_TOKEN;
-
-if (!slackBotToken) {
+const slackBotHostRoute = process.env.VCAP_APP_SLACK_BOT_HOST_ROUTE;
```

Comment:
The SLACK_BOT_HOST_ROUTE is only needed for running on bluemix, so there is no need put it in .env, or to check for it. If it's not in VCAP, we assume they are running local.